src/axom/mir/docs/sphinx/mir_algorithms.rst

```diff
@@ -0,0 +1,153 @@
+MIR Algorithms
+******************************************************
+
+The MIR component contains MIR algorithms that will take a Blueprint mesh as input,
+perform MIR on it, and output a new Blueprint mesh with the reconstructed output.
+A Blueprint mesh is contained in a ``conduit::Node`` and it follows the `Blueprint protocol <https://llnl-conduit.readthedocs.io/en/latest/blueprint_mesh.html>`_,
+which means the node contains specific items that describe the mesh coordinates, topology, fields, and materials.
+
+#######
+Inputs
+#######
+
+MIR algorithms are designed to accept a Conduit node containing various options that can
+influence how the algorithm operates. The MIR algorithm copies the options node to the memory
+space where it will be used.
+
++---------------------------------+------------------------------------------------------+
+| Option                          | Description                                          |
++=================================+======================================================+
+|``coordsetName: name``           | The name of the new coordset in the output mesh. If  |
+|                                 | it is not provided, the output coordset will have the|
+|                                 | same name as the input coordset.                     |
++---------------------------------+------------------------------------------------------+
+|``fields:``                      | The fields node lets the caller provide a list of    |
+|                                 | field names that will be processed and added to the  |
+|                                 | output mesh. The form is *currentName:newName*. If   |
+|                                 | the *fields* node is not given, the algorithm will   |
+|                                 | process all input fields. If the fields node is empty|
+|                                 | then no fields will be processed.                    |
++---------------------------------+------------------------------------------------------+
+| ``matset: name``                | A required string argument that specifies the name   |
+|                                 | of the matset that will be operated on.              |
++---------------------------------+------------------------------------------------------+
+| ``matsetName: name``            | An optional string argument that specifies the name  |
+|                                 | of the matset to create in the output. If the name   |
+|                                 | is not given, the output matset will have the same   |
+|                                 | name as the input matset.                            |
++---------------------------------+------------------------------------------------------+
+| ``originalElementsField: name`` | The name of the field in which to store the original |
+|                                 | elements map.                                        |
```

Comment:
From line 108, I think the default value is "originalElements".  If that's so, please note it here.

Reply:
Added.